test/client/api.client.spec.unit.ts

```diff
@@ -26,6 +25,15 @@ describe("apiclient company profile unit tests", () => {
         const company = await getCompanyProfile("00006400", mockUtils.ACCESS_TOKEN);
         expect(company).toEqual(expectedProfile);
     });
+
+    it("returns a 400 status code when company is not found", async () => {
+        mockGetCompanyProfile.mockResolvedValueOnce(notFoundSDKResponse);
+        try {
+            await getCompanyProfile("", mockUtils.ACCESS_TOKEN);
+        } catch (e) {
+            expect(e.status).toEqual(400);
+        }
```

Comment:
can you add `expect.assertions(1);` after the catch, that way we know that the exception expect was executed. Otherwise it might not throw an exception and the test will still pass.